website/pages/extending-theme.mdx

```diff
@@ -61,3 +61,22 @@ new Vue({
 }).$mount()
 ```
 
+### In Nuxt.js
+In your nuxt.config.js file, you can then add your custom theme object by passing it in the `extendTheme` option.
+
+```js
+// Import your customTheme object
+import customTheme from 'custom-theme.js'
+export default {
+  ...
+  modules: [
+    '@chakra-ui/nuxt',
+    '@nuxtjs/emotion'
+  ],
+  chakra: {
+      extendTheme: customTheme
+    }
+  }
+  ...
```

Comment:
Thanks for the contribution toward this. Do you think that it's probably better to reduce the amount of boiler plate in the `nuxt.config.js` code snippet here? So we'd only leave the chakra option.
```suggestion
  // ...
  chakra: {
      extendTheme: customTheme
    }
  }
  // ...
```

Reply:
Yes, it will be clear enough, but then I do think the last } has to be deleted too isn't it?

Reply:
Agreed on the reduced boilerplate, we already cover the setup in the nuxt.js section so I don't think it's necessary.

Reply:
@carwack Yup! Good catch with the closing brace. Was a typo. 👍🏽

Reply:
@codebender828 @dombavetta Thanks for the input! I did refactor the text. 😄